scripts/fieldbus/bench_test.sh

```
@@ -0,0 +1,378 @@
+#   OPENFDD_FIELDBUS_API_KEY=... scripts/bench_test.sh
+#   BENCH_MINUTES=30 BENCH_INTERVAL_SECS=60 scripts/bench_test.sh   # half-hour soak
+#   BENCH_PCAP=0 scripts/bench_test.sh                              # skip PCAP phases
+#
+# Haystack against Niagara requires sidecar started with:
+#   HAYSTACK_BASE_URL=https://192.168.204.11/haystack
+#   HAYSTACK_USER=open_fdd HAYSTACK_PASS=... HAYSTACK_AUTH_MODE=basic
+set -uo pipefail
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+# shellcheck source=scripts/bench_lib.sh
+source "$ROOT/scripts/bench_lib.sh"
```

Shellcheck (0.11.0)

[info] Not following: scripts/bench_lib.sh was not specified as input (see shellcheck -x).

(SC1091)